### PR TITLE
Add Simona card

### PR DESCRIPTION
**Pull request by SimonaPiz**
Add my card in index.html

### CODE REVIEW

**Review by alsonick — CHANGES_REQUESTED** [review]
Please replace the `*TEMPLATE*` text with either your name or username. Thanks

**Comment by SimonaPiz** [comment]
@alsonick  
> Please replace the `*TEMPLATE*` text with either your name or username. Thanks

I have replace it with my name. 👍

**Review by alsonick — CHANGES_REQUESTED** [review]
You need to replace the `*TEMPLATE*` text on line 140, please undo the changes you've made on line 108. Thanks

File: index.html [hunk 1]
```diff
@@ -105,7 +105,7 @@ <h2>A project for first-time contributions</h2>
                   </ul>
               </div>
           </div>
-          <!-- ________ *TEMPLATE* Contributor card END ________  -->
+          <!-- ________ Simona Pizio Contributor card END ________  -->
```
Comment (alsonick) [inline index.html:108]: Please undo the changes on line 108.

**Review by alsonick — CHANGES_REQUESTED** [review]

**Review by CandidDeer — APPROVED** [review]

**Comment by CandidDeer** [comment]
Thank you for your contribution. I have merged your PR and your card will be live in a moment.

**Pull request merged**